app/forms/utils/SessionKeys.scala

```diff
@@ -23,4 +23,6 @@ object SessionKeys {
   val chargeSummaryBackPage: String = "chargeSummaryBackPage"
 
   val summaryData: String = "summaryData"
+
+  val origin: String = "origin"
```

Comment:
if we want this here, we should remove from the trait.

Reply:
Deleted the trait one.